baseplate/server/runtime_monitor.py

```diff
@@ -272,10 +273,18 @@ def start(server_config: Dict[str, str], application: Any, pool: Pool) -> None:
         except Exception as exc:
             logger.info("monitoring.gc.refcycle disabled: %s", exc)
 
+    statsd_metrics_client = baseplate._metrics_client
```

Comment:
It _probably_ doesn't make a difference, but I feel like it makes more sense for this to be before any of the reporters are instantiated, _just in case_ it tries to report metrics in the `__init__()` functions

Reply:
I did it here in order to address the specific issue we see. We can move it if we see the issue elsewhere. What I was trying to avoid is context.metrics being set to a nil client. It wouldn't be ideal for large amounts of metrics to be flowing through this client. This is a shim for Baseplate internals only, not for applications to use.